testing/util.go

```diff
@@ -50,3 +51,17 @@ func RunWithTimeout(t *testing.T, d time.Duration, f func()) {
 	}()
 	<-ch
 }
+
+// TrimTestArgs finds the first -- in os.Args and trim all args before that
+func TrimTestArgs() {
```

Comment:
Yeah I wish - for some reason it trims a little too much off.  I could investigate more if you like?